docs/src/app/components/pages/components/Slider/ExampleValue.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Slider from 'material-ui/Slider';
+
+export default class SliderExampleValue extends React.Component {
+
+  state = {
+    firstSlider: 0.5,
+    secondSlider: 50,
+  }
+
+  handleSlider(key, value) {
+    this.setState({[key]: value});
+  }
+
+  render() {
+    return (
+      <div>
+        <Slider
+          defaultValue={0.5}
+          value={this.state.firstSlider}
+          onChange={(event, value) => this.handleSlider('firstSlider', value)}
```

Comment:
> there's no way to actually get the value without putting a function in the render method

It depends of what you mean but `putting`.
We do need to give a function for the `onChange` property. However, we don't have to create a new function at each render with a `bind`. 
Look at the example I have linked in the previous comment. That's working fine with an arrow function as a class property.